BrainCloudClient/src/BrainCloud/Client/BrainCloud/BrainCloudWrapper.cs

```diff
@@ -89,14 +89,41 @@ public class BrainCloudWrapper
     public static string AUTHENTICATION_ANONYMOUS = "anonymous";
 
 
+    // Name of this wrapper instance. Used for data loading
+    public string _wrapperName = "";
+    
+    /// <summary>
+    /// Create the brainCloud Wrapper, which has utility helpers for using the brainCloud API
+    /// </summary>
     public BrainCloudWrapper()
     {
-        _client = BrainCloudClient.Get();
+        client = new BrainCloudClient();
+    }
+    
+    /// <summary>
+    /// Create the brainCloud Wrapper, which has utility helpers for using the brainCloud API
+    /// </summary>
+    /// <param name="bcClient">set wrapper with a specfic brainCloud client</param>
+    private BrainCloudWrapper(BrainCloudClient bcClient)
+    {
+        client = bcClient;
+    }
+    
+    /// <summary>
+    /// Create the brainCloud Wrapper, which has utility helpers for using the brainCloud API
+    /// </summary>
+    /// <param name="wrapperName">string value used to differentiate saved wrapper data</param>
+    public BrainCloudWrapper(string wrapperName)
+    {
+        client = new BrainCloudClient();
+        _wrapperName = wrapperName;
     }
 
+    [Obsolete("Use of the *singleton* has been deprecated. We recommend that you create your own *variable* to hold an instance of the brainCloudWrapper. Explanation here: http://getbraincloud.com/blog")]

```

Comment:
I'd try to group all the obsolete functions together... so that its not mixed within the file..